### DISCUSSION TITLE
How kong report  trace to jaeger??

### DISCUSSION THREAD

**Discussion by zhongshan365** [Kong/kong#8050] [Help] [answered]
I want to report trace to jaeger instead of zipkin. I saw that there is a plugin https://github.com/Kong/opentracing-lua .
But I donot how to use it? There is just a command 'luarocks install opentracing'.
where to config jaeger address?

Can someone help me? Thanks

**Comment by kikito** [comment, marked as answer]
Good day, that library is not used any more.

The Zipkin plugin can be used with Jaeger.

https://docs.konghq.com/hub/kong-inc/zipkin

Notice that the `header_type` and `default_headter_type` must be changed to `jaeger`
